### PR TITLE
merge cli command serialpassthrough from cleanflight

### PR DESCRIPTION
**Pull request by metalnow**
reference from cleanflight. just copy some code to implement.
I tested it and looks working. but I don't know if there are some side-effect. 
please help testing. thanks.

### CODE REVIEW

File: src/main/fc/cli.c [hunk 1]
```diff
@@ -1759,6 +1759,81 @@ static void cliSerial(char *cmdline)
     memcpy(currentConfig, &portConfig, sizeof(portConfig));
 }
 
+#ifndef SKIP_SERIAL_PASSTHROUGH
```
Comment (digitalentity) [inline src/main/fc/cli.c]: For consistency with other features this should be renamed to `USE_SERIAL_PASSTHROUGH`.

**Review by digitalentity — CHANGES_REQUESTED** [review]
This feature should only be made available for targets with enough flash - `>128K` section in `common.h`

File: src/main/fc/cli.c (hunk 1 above)
Comment (metalnow) [reply src/main/fc/cli.c]: OK,,, Should I update all SKIP_SERIAL_PASSTHROUGH to USE_SERIAL_PASSTHROUGH? I search whole files and found one in line 499 of io/serial.c .

Comment (digitalentity) [reply src/main/fc/cli.c]: Yes, I think we should change to `#ifdef USE_SERIAL_PASSTHROUGH` and enable that feature in `common.h` only for targets with enough flash

File: src/main/target/CJMCU/target.h [hunk 2]
```diff
@@ -100,7 +100,7 @@
 
 #define BRUSHED_MOTORS
 #define DEFAULT_FEATURES        FEATURE_MOTOR_STOP
-#define SKIP_SERIAL_PASSTHROUGH
+#undef USE_SERIAL_PASSTHROUGH
```
Comment (digitalentity) [inline src/main/target/CJMCU/target.h:103]: this is not really necessary, but won't hurt

File: src/main/target/CRAZEPONYMINI/target.h [hunk 3]
```diff
@@ -69,7 +69,7 @@
 
 #undef USE_RX_MSP
 #define SKIP_INFLIGHT_ADJUSTMENTS
-#define SKIP_SERIAL_PASSTHROUGH
+#undef USE_SERIAL_PASSTHROUGH
```
Comment (digitalentity) [inline src/main/target/CRAZEPONYMINI/target.h:72]: this is not really necessary, but won't hurt

**Review by digitalentity — APPROVED** [review]

File: src/main/fc/cli.c (hunk 1 above)
Comment (metalnow) [reply src/main/fc/cli.c]: Thank you for reviewing. Hope i am doing right.

File: src/main/target/CJMCU/target.h (hunk 2 above)
Comment (metalnow) [reply src/main/target/CJMCU/target.h:103]: OK, i just leave it there.

**Comment by digitalentity** [comment]
Build is still failing - now missing `#ifdef` for the feature in `cli.c`

**Comment by metalnow** [comment]
Sorry, my bad. I should be more careful.

**Comment by oleost** [comment]
Should CLI.md be updated to reflect this PR?

**Comment by metalnow** [comment]
@oleost  it won't. I have not udpate CLI.md yet. after pr approved, I will create another pr to update CLI.md for serailpassthrough function.

**Comment by digitalentity** [comment]
We need to test this in a few different use-cases, then it's ok to be merged

**Pull request merged**